.cow.json

```diff
@@ -19,6 +19,7 @@
   "upgrade-only": [
     "dnadesign/silverstripe-elemental-subsites",
     "dnadesign/silverstripe-elemental-userforms",
-    "undefinedoffset/sortablegridfield"
+    "undefinedoffset/sortablegridfield",
+    "silverstripe/subsites"
```

Comment:
Subsite is now meant to be an upgrade only module.

Reply:
perhaps we should make it so in the `.cow.json` config then?

Reply:
Not sure I understand ... isn't what I'm doing right here?

Reply:
I'm only saying we could mark the module as "upgrade-only" so that we don't release it together with kitchen-sink anymore

Reply:
let's go with it as `not upgrade-only`. that's a safer default.